### PR TITLE
Update disqus_comments.html

### PR DESCRIPTION
**Pull request by jyyo0n**
Here We need to append **https://** prefix to this code.

https://help.disqus.com/en/articles/1717301-i-m-receiving-the-message-we-were-unable-to-load-disqus

### CODE REVIEW

**Comment by rohitjain00** [comment]
Are you facing any issue here? I'm using this template only on my personal portfolio, seems to be working fine.

Can you add some more details for this?

https://therohitjain.com/jekyll/deployment/2020/12/29/Hosting-Jekyll-blogs-on-github.html

**Comment by jyyo0n** [comment]
First of all, the template is so pretty that I'm going to try it too :D
Anyway, your site code looks like below, but my site code is missing **https://** so I have to add it.

`<script type="529db15b6ffbecec7eaa1cc5-text/javascript">
var disqus_config=function(){
this.page.url="https://therohitjain.com/jekyll/deployment/2020/12/29/Hosting-Jekyll-blogs-on-github.html",
this.page.identifier="https://therohitjain.com/jekyll/deployment/2020/12/29/Hosting-Jekyll-blogs-on-github.html"};`

File: _includes/disqus_comments.html [hunk 1]
```diff
@@ -11,4 +11,4 @@
     (d.head || d.body).appendChild(s);
   })();
 </script>
-<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>
\ No newline at end of file
+<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>
```
Comment (rohitjain00) [inline _includes/disqus_comments.html:14]: Any reason for this change?

Comment (rohitjain00) [reply _includes/disqus_comments.html:14]: Is this required, if not, can you update the commit to remove this change?

Comment (jyyo0n) [reply _includes/disqus_comments.html:14]: I didn't touch this code, but it says it's changed 😢

Comment (rohitjain00) [reply _includes/disqus_comments.html:14]: Can you update such that this is not reflected here?

**Pull request closed without merging**